cosmo_tester/framework/test_hosts.py

```diff
@@ -868,6 +868,28 @@ def _set_image_details(self):
                 self._test_config['test_os_usernames'][username_key]
             )
 
+    def prepare_xfsdump_volume(self):
+        self._logger.info('Preparing XFS volume for XFS-dump')
+        with self.ssh() as fabric_ssh:
+            # Identify the still not partitioned volume
+            result = fabric_ssh.run(
+                "lsblk -o NAME,TYPE -ds | awk '$2 == \"disk\" {print $1}'")
+            self._logger.info('>>>> Disk name = `%s`', result.stdout())
+
+            fabric_ssh.run(
+                "export XFSVOL=$(lsblk -o NAME,TYPE -ds "
+                "| awk '$2 == \"disk\" {print $1}')")
+
+#             result = fabric_ssh.run(
```

Comment:
Please remove the commented code.

Reply:
Switching to reviewing as one block, not by commit- will re-raise comments as required.